requirements/frozen.txt

```diff
@@ -199,4 +199,4 @@ wrapanapi==2.3.1
 wrapt==1.10.10
 xmltodict==0.11.0
 yaycl==0.2.0
-yaycl-crypt==0.2.0
+yaycl-crypt==0.2.0
```

Comment:
This dependency already exists in [frozen.txt](https://github.com/ManageIQ/integration_tests/blob/master/requirements/frozen.txt#L202)